### PR TITLE
Update ingress v1 and v1beta

### PR DESCRIPTION
**Pull request by scoyle391**

### CODE REVIEW

File: reference/ambassador-with-gke.md [hunk 1]
```diff
@@ -2,20 +2,23 @@
...
+
+In this setup, the ingress resource creates two forwarding rules, one for HTTP and one for HTTPS. The HTTPS
+forwarding rule has the SSL certificates attached. In addition, one backend service will be created to point to
 a list of instance groups at a static port. This will be the NodePort of the Ambassador service. 
 
-With this setup the load balancer terminates HTTPS and then directs the traffic to the Ambassador service 
+With this setup, the load balancer terminates HTTPS and then directs the traffic to the Ambassador service 
 via the NodePort. Ambassador is then doing all the routing to the other internal/external services. 
 
+Currently, the Ambassador Edge Stack supports Kubernetes the `Ingress` apiVersion `v1` and `v1beta1`.
```
Comment (alexgervais) [inline reference/ambassador-with-gke.md]: Do we want to mention Ambassador OSS also supports it?

File: reference/ambassador-with-gke.md [hunk 2]
```diff
@@ -81,7 +84,20 @@ spec:
     servicePort: 8080
 ```
 
-Now let's connect the other service from the tutorial to ambassador by specifying a mapping:
+Or 
+
+```yaml
+apiVersion: extensions/v1
```
Comment (alexgervais) [inline reference/ambassador-with-gke.md]: This sample is out of date with the `v1` API.
@alexgervais to provide a good and valid example.

File: reference/ambassador-with-gke.md [hunk 3]
```diff
@@ -115,9 +133,24 @@ spec:
...
+    networking.gke.io/managed-certificates: www-example-com
+spec:
+  backend:
+    serviceName: ambassador
+    servicePort: 8080
+```
+ or 
+
+```yaml
+apiVersion: extensions/v1
```
Comment (alexgervais) [inline reference/ambassador-with-gke.md]: This sample is out of date with the `v1` API.
@alexgervais to provide a good and valid example.

File: reference/core/ingress-controller.md [hunk 4]
```diff
@@ -107,7 +107,7 @@ For example, this `Ingress` resource
 
 ```yaml
 ---
-apiVersion: extensions/v1beta1
+apiVersion: extensions/v1
```
Comment (alexgervais) [inline reference/core/ingress-controller.md]: This sample is out of date with the `v1` API.
@alexgervais to provide a good and valid example.

File: reference/core/ingress-controller.md [hunk 5]
```diff
@@ -140,7 +140,7 @@ This means that the following YAML:
...
```
Comment (alexgervais) [inline reference/core/ingress-controller.md]: This sample is out of date with the `v1` API.
@alexgervais to provide a good and valid example.

File: reference/core/ingress-controller.md [hunk 6]
```diff
@@ -171,7 +171,7 @@ will set up the Ambassador Edge Stack to do canary routing where 50% of the traf
 An `Ingress` resource must provide at least some routes or a [default backend](https://kubernetes.io/docs/concepts/services-networking/ingress/#default-backend). The default backend provides for a simple way to direct all traffic to some upstream service:
 
 ```yaml
-apiVersion: extensions/v1beta1
+apiVersion: extensions/v1
```
Comment (alexgervais) [inline reference/core/ingress-controller.md]: This sample is out of date with the `v1` API.
@alexgervais to provide a good and valid example.

File: reference/core/ingress-controller.md [hunk 7]
```diff
@@ -200,7 +200,7 @@ spec:
 
 ```yaml
 ---
-apiVersion: extensions/v1beta1
+apiVersion: extensions/v1
```
Comment (alexgervais) [inline reference/core/ingress-controller.md]: This sample is out of date with the `v1` API.
@alexgervais to provide a good and valid example.

File: reference/core/ingress-controller.md [hunk 8]
```diff
@@ -255,7 +255,7 @@ Read more from Kubernetes [here](https://kubernetes.io/docs/concepts/services-ne
 ### TLS Termination
 
 ```yaml
-apiVersion: extensions/v1beta1
+apiVersion: extensions/v1
```
Comment (alexgervais) [inline reference/core/ingress-controller.md]: This sample is out of date with the `v1` API.
@alexgervais to provide a good and valid example.

File: reference/core/ingress-controller.md [hunk 9]
```diff
@@ -14,7 +14,7 @@ If you're new to the Ambassador Edge Stack and to Kubernetes, we'd recommend you
 
    Starting in Kubernetes 1.14, the `Ingress` was added to the new `networking.k8s.io` api.
 
-   **Note:** If you are using 1.14 and above, it is recommended to use `apiVersion: networking.k8s.io/v1beta1` when defining `Ingresses`. Since both are still supported in all 1.14+ versions of Kubernetes, this document will use `extensions/v1beta1` for compatibility reasons.
+   **Note:** If you are using 1.14 and above, Ambassador supports both `v1` and `v1beta1` when defining `Ingresses`. Since both are still supported in all 1.14+ versions of Kubernetes, this document will use `extensions/v1` for compatibility reasons, but you can still use `v1beta1`.
```
Comment (alexgervais) [inline reference/core/ingress-controller.md]: It should read something like:
> If you are using **Kubernetes 1.19** and above, Ambassador supports `apiVersion: networking.k8s.io/v1`. For compatibility reasons, Ambassador also supports the deprecated `apiVersion: networking.k8s.io/v1beta1` and `apiVersion: extensions/v1beta1` when defining `Ingresses` if you are using an older version of Kubernetes.

The important point is mentioning `v1` support in Kubernetes 1.19 and the deprecation of `v1beta1`, although we keep supporting it for older Kubernetes versions.

**Comment by concaf** [comment]
FYI: This repository will be archived shortly. Please send this PR against https://github.com/datawire/ambassador/tree/master/docs.

**Comment by alexgervais** [comment]
Closing. Will target datawire/ambassador.git

**Pull request closed without merging**